src/cc/frontends/clang/b_frontend_action.cc

```diff
@@ -354,8 +354,14 @@ bool BTypeVisitor::VisitCallExpr(CallExpr *Call) {
           txt += ", bpf_get_smp_processor_id(), " + args_other + ")";
         } else {
           if (memb_name == "lookup") {
```

Comment:
why override .lookup ?
programs can do lookup on stackmap and it will return a set of ip addresses.
I was planning to use 'bounded loop' to do a search in there.
@brendangregg mentioned that it would be nice to filter based on ip address or range of addresses.
so that's what stackmap.lookup() from bpf program is for.